pkg/monitoring/alloy/configmap.go

```diff
@@ -60,7 +60,7 @@ func (a *Service) GenerateAlloyMonitoringConfigMapData(ctx context.Context, curr
 	}
 
 	// Compute the number of shards based on the number of series.
-	query := fmt.Sprintf(`sum(max_over_time((sum(prometheus_remote_write_wal_storage_active_series{cluster_id="%s", component_id="prometheus.remote_write.default", service="%s"})by(pod))[6h:1h]))`, cluster.Name, commonmonitoring.AlloyMonitoringAgentAppName)
+	query := fmt.Sprintf(`sum(max_over_time((sum(prometheus_remote_write_wal_storage_active_series{cluster_id="%s", component_id="prometheus.remote_write.giantswarm", service="%s"})by(pod))[6h:1h]))`, cluster.Name, commonmonitoring.AlloyMonitoringAgentAppName)
```

Comment:
I think we should make the metric tenant agnostic because alloy should scaled according to the amount of metrics ingested in total and not only for the giantswarm tenant.

Reply:
https://github.com/giantswarm/observability-operator/pull/329